src/java.base/share/classes/java/lang/Object.java

```diff
@@ -127,15 +127,26 @@ public Object() {}
      * <li>For any non-null reference value {@code x},
      *     {@code x.equals(null)} should return {@code false}.
      * </ul>
+     *
      * <p>
+     * An equivalence relation partitions the elements it operates on
+     * into <i>equivalence classes</i>; all the members of an
+     * equivalence class are equal to each other. The equal objects
+     * are substitutable for each other, at least for some purposes.
```

Comment:
Since the preceding sentence mentions the members of an equivalence class, it might read better to say "Members are substitutable..." or possibly "Members of an equivalence class are substitutable...."

Reply:
Phrasing updated to:

"Members of an equivalence class are substitutable for each other, at least for some purposes."